src/common/framework/params.ts

```diff
@@ -33,7 +33,7 @@ export function params(): ParamsBuilder<{}> {
   return new ParamsBuilder();
 }
 
-class ParamsBuilder<A extends {}> implements ParamSpecIterable {
+export class ParamsBuilder<A extends {}> implements ParamSpecIterable {
```

Comment:
Compiles and lints fine without this, but VSCode complains?

> Return type of public static method from exported class has or is using name 'ParamsBuilder' from external module "src/common/framework/params" but cannot be named.ts (4050)

Reply:
I didn't see this in linting. Getting vscode to sync exactly with the linting rules is tricky. If it doesn't affect linting I'd slightly prefer to not export this (but it doesn't really matter).

Reply:
Okay, removed the export.

Reply:
I have two extensions: Prettier and ESLint (removed tslint). Together they seem to work (Prettier is needed otherwise ESLint screws up indentation) and I don't see that warning